### PR TITLE
list weeks & parts in readme

### PR DESCRIPTION
**Pull request by jsms90**
+ move definitions of "curriculum", "week" and "part" from `vision.md` to the readme
+ list all weeks
+ link to the open issues under each week's label
+ list all morning challenges & workshops under each week

Relates #216 #155 #137

### CODE REVIEW

**Comment by jsms90** [comment]
@des-des Agreed, this might not be the best format.

But if the `master-reference` is supposed to unable people to create their own curriculum from various parts, you need to list those parts somewhere with an actual explanation of when/why you might use them.

I think it's too much to expect someone to look through the detail of every single week which exists, in order to decide which ones to use. Something a little like [this](https://github.com/foundersandcoders/master-reference/blob/master/coursebook/fragments/product-build/tech-choices.md)

**Comment by des-des** [comment]
@jsms90 sorry I have been so slow on this, am easy.

**Comment by jsms90** [comment]
@des-des I'm genuinely interested in your take on this. If you can think of a better way, I'm all ears!

**Comment by des-des** [comment]
@jsms90 I like what have here, I think it is a good document. I think we can revisit in 4 months.

File: README.md [hunk 1]
```diff
@@ -4,4 +4,108 @@
...
+
+### Hapi 2
+[Open issues](https://github.com/foundersandcoders/master-reference/issues?q=is%3Aopen+is%3Aissue+label%3Aweek-hapi-auth)
+
+#### Workshops
+* [OAuth workshop](https://github.com/foundersandcoders/oauth)
+* [hapi Authentication workshop](https://github.com/foundersandcoders/hapi-authentication-workshop)
+* [JWT workshop](https://github.com/foundersandcoders/jwt_workshop)
+
+### Express
```
Comment (eliascodes) [inline README.md]: If we're adding express, should we also add the new auth-week?

Comment (jsms90) [inline README.md]: It's on line 87, before the frameworks.

**Comment by jsms90** [comment]
@eliascodes But I will add the old OAuth

File: README.md (hunk 1 above)
Comment (eliascodes) [inline README.md]: Sorry, didn't see it, all good!

**Pull request merged**